CONTRIBUTING.md

```diff
@@ -39,6 +39,53 @@ Anything contained within the _src_ folder is meant to be merged from the upstre
 
 There is no intention to support any mkdocs extensions in this theme because this theme is designed for the Sphinx documentation generator. You will often find that most of the extensions available for mkdocs are natively implemented in Sphinx or available in the form of a Sphinx extension.
 
+## Building this project's docs
+
+An example of instructions for Ubuntu 24.04:
+
+```
+$ lsb_release -a
+Distributor ID:  Ubuntu
+Description:     Ubuntu 24.04.3 LTS
+Release:         24.04
+Codename:        noble
+```
+
+Fetch source code:
+
+```
+$ git clone git@github.com:jbms/sphinx-immaterial
+$ cd sphinx-immaterial/
+```
+
```

Comment:
I think this would be implied for contributors. In fact, it may be misleading if working from a fork toward a contribution. This can also be removed.